src/work-allocation-2/components/available-tasks-filter/available-tasks-filter.component.scss

```diff
@@ -0,0 +1,17 @@
+:host ::ng-deep .available-tasks_location-filter {
+  .xui-checkbox-list {
+    height: 200px;
+    // border-bottom: solid 1px #b1b4b6;
+    padding-bottom: 20px;
+    .select-all {
+      margin-bottom: auto;
+    }
+    .checkbox-items {
+      margin-left: -10px;
+      padding-left: 10px;
+    }
+  }
+}
+.govuk-section-break {
+  height: auto;
+}
```

Comment:
Again no new line at the end of file.

Reply:
New line is required for .ts file. Not needed for html and scss